user-guide/Advanced_Modules/User Defined APIs/User_Defined_APIs.md

```diff
@@ -0,0 +1,43 @@
+---
+uid: UD_APIs
+---
+
+# User Defined APIs
+
+> [!WARNING]
+> The current feature is in preview and is not fully released yet. This feature should not be used in any staging or production environment.
+
+The User Defined APIs feature allows users to define API calls that will be made available on the DataMiner agents that host the [UserDefinableApiEndpoint extension module](xref:UD_APIs_UserDefinableApiEndpoint). These APIs can be secured using API tokens that can be generated on the fly & linked to the API definitions.
+
+## Cube
+
+In System Center in Cube, you can manage your APIs and tokens in the User-Defined APIs tab.
+
+> [!WARNING]
+>- This tab will replace the obsolete API deployment tab in the future, so make sure to move your old APIs from API deployment to the new feature. How to create APIs from existing scripts is explained on the [using an existing script page](xref:UD_APIs_Using_existing_scripts).
+>- The UI is not visible by default as the feature is not released yet.
+
+## What are the different objects?
+
+|Name          |Description|
+|--------------|-----------|
+|ApiToken      |Defines a secret string + metadata that can be used to access an API.|
+|ApiDefinition |Defines the API. Including what tokens have access, the URL route and what action should be triggered.|
+
+## Client Test UI
```

Comment:
Open for discussion: should we keep this part about Client test tool UI?

Reply:
I'm in favor of keeping it. You already wrote it and it's a small one to review